advocate_europe/templates/email_base.html

```diff
@@ -10,34 +10,66 @@
             padding: 0;
             min-width: 100% !important;
         }
+
+        .main {
+          display: flex;
```

Comment:
Since this is an HTML-Email you should do web stuff like you are in the late 90s. Using flexbox is definitely to risky I would say and some [random folks](https://litmus.com/community/discussions/1500-using-flexbox-in-an-email) on the internet agree :smile:

Reply:
I removed the flex-voodoo in a006042.